### PR TITLE
Fixed parallel-jobs for loadBalancer case to take the one passed in the command

### LINKED ISSUE #744: [Load Balancer] --parallel-jobs is broken
Even if we provide a different value for the flag `--parallel-jobs` in case of load balancer, the default value of 6 is used.

Image

Image

### PR DESCRIPTION
**Pull request by priyanshi-yb**
Fixes #744

### CODE REVIEW

**Review by amit-yb — APPROVED** [review]
LGTM. Please test the fix before merging it.

File: yb-voyager/cmd/importData.go [hunk 1]
```diff
@@ -197,8 +197,10 @@ func getYBServers() []*tgtdb.Target {
 
 	if loadBalancerUsed { // if load balancer is used no need to check direct connectivity
 		utils.PrintAndLog(LB_WARN_MSG)
-		parallelImportJobs = 2 * len(targets)
-		utils.PrintAndLog("Using %d parallel jobs by default. Use --parallel-jobs to specify a custom value", parallelImportJobs)
+		if parallelImportJobs == -1 {
+			parallelImportJobs = 2 * len(targets)
+			utils.PrintAndLog("Using %d parallel jobs by default. Use --parallel-jobs to specify a custom value", parallelImportJobs)
```
Comment (sanyamsinghal) [inline yb-voyager/cmd/importData.go:202]: Nit: Remove this print line from here
Also remove similar line from line 328
Add this line just once at line 330

**Review by sanyamsinghal — APPROVED** [review]
LGTM, provided a minor comment

File: yb-voyager/cmd/importData.go (hunk 1 above)
Comment (priyanshi-yb) [reply yb-voyager/cmd/importData.go:202]: this print should be only in the case of the default parallel-jobs where user is not using --parallel-jobs which is this condition - (parallelImportJobs == -1), how can we remove it from both the places add at some other?

Comment (priyanshi-yb) [reply yb-voyager/cmd/importData.go:202]: resolved, over discussion.

**Pull request merged**